app/src/main/res/layout/view_pledge_layout.xml

```diff
@@ -165,7 +165,31 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:paddingBottom="@dimen/grid_3"/>
+        </LinearLayout>
+
+        <LinearLayout
+          android:id="@+id/view_pledge_estimated_delivery_section"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:orientation="vertical">
+
+          <include layout="@layout/horizontal_line_1dp_view"/>
+
+          <TextView
+            style="@style/SubheadPrimaryMedium"
+            android:text="Estimated Delivery"
```

Comment:
think lint is just yelling at this hardcoded text, should use an `@string/` resource here to appease and be good i18n-ers pls